src/Components/URLParams.tsx

```diff
@@ -0,0 +1,17 @@
+import '../CSS/URLParams.css';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import PokeDetails from './PokeDetails.tsx';
+
+const URLParams = () => {
+
+  const id = useParams().id;
```

Comment:
This is minimal enough I would probably just leave it in your PokeDetails component (unless you're planning on adding more to this!)